src/context/banners.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useContext } from "react";
+import { BannerTypeKeys } from "types/banner";
+import { v4 as uuid } from "uuid";
+
+export interface BannerObj {
+  id: string;
+  type: BannerTypeKeys;
+  message: string;
+}
+type AddBanner = (message: string) => void;
+interface DispatchBanner {
+  error: AddBanner;
+  success: AddBanner;
+  warning: AddBanner;
+  info: AddBanner;
+  remove: (bannerId: string) => void;
+  clearAll: () => void;
+}
+
+const BannerDispatchContext = React.createContext<DispatchBanner | null>(null);
+const BannerStateContext = React.createContext<BannerObj[] | null>(null);
+
+export const BannerContextProvider: React.FC = ({ children }) => {
+  const [banners, setBanners] = useState<BannerObj[]>([]);
+
+  const addBanner = (type: BannerTypeKeys, message: string) => {
```

Comment:
Lara had some ideas about the ordering of banner priority. She wants the warning banner to always have priority. She mentioned placing it in the wiki but i dont think that was made yet.

Reply:
done